control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -153,31 +162,27 @@ func (r *GatewayController) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, err
 	}
 
-	// fetch all consul objects from cache
+	// fetch the rest of the consul objects from cache
 	consulServices := r.getConsulServices(consulKey)
 	consulGateway := r.getConsulGateway(consulKey)
-	consulHTTPRoutes := r.getConsulHTTPRoutes(consulKey, resources)
-	consulTCPRoutes := r.getConsulTCPRoutes(consulKey, resources)
-	consulInlineCertificates := r.getConsulInlineCertificates()
+	r.fetchConsulHTTPRoutes(consulKey, resources)
+	r.fetchConsulTCPRoutes(consulKey, resources)
 
 	binder := binding.NewBinder(binding.BinderConfig{
-		Logger:                   log,
-		Translator:               r.Translator,
-		ControllerName:           GatewayClassControllerName,
-		Namespaces:               namespaces,
-		GatewayClassConfig:       gatewayClassConfig,
-		GatewayClass:             gatewayClass,
-		Gateway:                  gateway,
-		Pods:                     pods,
-		Service:                  service,
-		HTTPRoutes:               httpRoutes,
-		TCPRoutes:                tcpRoutes,
-		Resources:                resources,
-		ConsulGateway:            consulGateway,
-		ConsulHTTPRoutes:         consulHTTPRoutes,
-		ConsulTCPRoutes:          consulTCPRoutes,
-		ConsulInlineCertificates: consulInlineCertificates,
```

Comment:
dropped this and the two route fields above since they weren't used anywhere and a solely tracked by the resources map